tests/components/folder_watcher/test_init.py

```diff
@@ -20,7 +20,7 @@
 async def test_valid_path_setup(hass):
     """Test that a valid path is setup."""
     cwd = os.path.join(os.path.dirname(__file__))
-    hass.config.whitelist_external_dirs = set((cwd))
+    hass.config.whitelist_external_dirs = set(cwd)
```

Comment:
The change is equivalent, but... is this test broken to start with? E.g. if the path was "/a/b/c":

```
>>> set(("/etc/something/something.yaml"))
set(['a', 'o', 'c', 'e', 'g', 'i', 'h', 'm', 'l', '/', 'n', 's', 't', 'y', '.'])
```

Reply:
It actually is, however, in this case, it is just testing if the component setup passes...

Reply:
Adjusted the test to deal with that, thanks! 👍

Reply:
I bet someone missed a comma there. Defining single item tuples is a pain.

Reply:
One more reason to use literals ;)